tutorials/tts/Inference_ModelSelect.ipynb

```diff
@@ -49,7 +49,7 @@
     "BRANCH = 'main'\n",
     "# # If you're using Google Colab and not running locally, uncomment and run this cell.\n",
     "# !apt-get install sox libsndfile1 ffmpeg\n",
-    "# !pip install wget unidecode\n",
+    "# !pip install wget unidecode pynini==2.1.4\n",
```

Comment:
Let's just add pynini to the tts_requirements

Reply:
Fixed.